tests/foreman/ui/test_user.py

```diff
@@ -17,8 +17,10 @@
 """
 
 import random
+
 from fauxfactory import gen_string
 from nailgun import entities
+
```

Comment:
It's automatic generated by Pycharm. I think it is following the rule of separating groups of imports, according to pep 8: https://www.python.org/dev/peps/pep-0008/#imports. But I am going to change it and config Pycharm ;)